src/data/markdown/docs/40 xk6-disruptor/03 API/01 Faults/01 Grpc.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Grpc'
```

Comment:
I can't start a review yet, but my first comment is about capitalization.

It looks like the framework uses `gRPC`. Shouldn't we follow their capitalization?

https://grpc.io/about/

Reply:
Good catch. Fixed.